### ISSUE TITLE
Looking for Maintainers!

### ISSUE THREAD

**Issue by dergigi** [nostr-resources/nostr-resources.github.io#62]
This site could be way better (and up-to-date) than it is. If you're interested in maintaining it, please leave a comment in this issue, explaining why you would be a good fit.

**Comment by indramanohar** [comment]
Are you still looking for maintainers? If so, I would like to discuss and see what kind of maintainer you are looking for? A bit about me, I'm into Software development for the past 17 years and have experience developing web application using tech stack like ReactJS, .net core, Web API, SQL, Javascript and many more tools and libraries. Let me know if you think we can work it out.

**Comment by dergigi** [comment]
@indramanohar yes, I'm still looking! I guess the main criteria is that the maintainer really knows their shit when it comes to bitcoin. Are you qualified in that sense?

**Comment by indramanohar** [comment]
I’m not exactly sure what you mean by bitcoin context, if you can may be
give a sample task and see if I’m able to get it to work, it may be a good
catalyst for both of us to figure out if it works.

Manohar
"May you live every day of your life."

**Comment by Dawn11111111** [comment]
Going through, I've hit a dilemma. I don't know the proper etiquettte for how long something should be down or gone before it is removed or replaced. I have a list and some things that could be added, but I don't want to waste your time. Would you be able to drop me a little guidance on that?

**Comment by ButtercupRobrts** [comment]
Hi @dergigi what else is required of maintainers beside knowing a about nostr and bitcoin?

**Comment by Dawn11111111** [comment]
Thank you, @dergigi :) I really enjoy doing it. 
@ButtercupRobrts I'm happy to explain how I have been approaching this, if you need some ideas. If you want to chat about it, I do use Nostr DMs or you can just mention me in a note: https://njump.me/npub1cgcwm56v5hyrrzl5ty4vq4kdud63n5u4czgycdl2r3jshzk55ufqe52ndy

**Comment by dergigi** [comment]
It is! I'll close this issue now. 

Thank you for taking such good care of it @Dawn11111111 🙏🧡
